src/modules/Security.rb

```diff
@@ -731,35 +733,30 @@ def Import(settings)
     # (For use by autoinstallation.)
     # @return [Hash] Dumped settings (later acceptable by Import ())
     def Export
-      Builtins.eval(@Settings)
+      @Settings if @Settings.is_a?(Hash)
```

Comment:
well, here is bug, that eval do copy. Also check for hash is not needed, so it should look like
```
deep_copy(@Settings)
```